Modix.Bot/Modules/PingModule.cs

```diff
@@ -1,12 +1,20 @@
 ﻿using System.Threading.Tasks;
 using Discord.Commands;
+using Discord.WebSocket;
 
 namespace Modix.Modules
 {
     public class PingModule : ModuleBase
     {
+        public PingModule(DiscordSocketClient discordClient)
+        {
+            DiscordClient = discordClient;
+        }
+
         [Command("ping")]
         public Task Ping() =>
-            ReplyAsync("Pong!");
+            ReplyAsync($"Pong! ({DiscordClient.Latency} ms)");
+
+        protected DiscordSocketClient DiscordClient { get; }
```

Comment:
Huh? Why `protected`?

Reply:
Why not?

Reply:
What are you expecting to inherit from `PingModule`?

Reply:
There's no reason to expose this field

Reply:
In that case, would you recommend sealing the class and making the property/field private?

Reply:
Yes, I realize I'm being super nitpicky here, but there's not a lot to pick on when the PR is 10 lines 😬

Reply:
Updated